### ISSUE TITLE
ARM 0.9.5 :: No documentation for -OutputObjectFormat parameter (P2)

### ISSUE THREAD

**Issue by pcgeek86** [Azure/azure-powershell#638]
# Symptom

The ARM PowerShell module has several commands that throw a warning about the `-OutputObjectFormat` parameter. There is no documentation that explains what this parameter is for, and why the user should care about it.

![image](https://cloud.githubusercontent.com/assets/466713/8813739/cf3ab1ca-2fc2-11e5-9ab4-bc202d770b3e.png)

![image](https://cloud.githubusercontent.com/assets/466713/8813753/f9e64d94-2fc2-11e5-9654-b20d1ba295c3.png)

# Expected Results

There should be documentation included to describe what the `-OutputObjectFormat` parameter is for, why it exists, and when to use it.

Cheers,
Trevor Sullivan
Microsoft MVP: PowerShell

**Comment by devigned** [comment]
It's been tagged and it's being triaged. Thank you for the issue report.

Perhaps we can have a little more of the conversation around this in the gh issue rather than email. I think it would be health to have that discussion.

**Comment by pcgeek86** [comment]
@devigned Happy to, but I think the actionable items are pretty clear. 

1. Create a documentation page on Azure.microsoft.com
2. Implement appropriate portions of documentation into the built-in PowerShell help

To be completely honest with you David, there are simply too many documentation and functional gaps, and implementation inconsistencies, to spend a long time hashing each one out in a lengthy discussion. These kinds of things need to be "nipped in the bud," so to speak, so we can move on to more important topics.

I spend so much time reporting documentation and functional issues, using a variety of mediums, that it is having a very measurable and serious impact on my ability to produce work for my customers, much less for myself.

Cheers,
Trevor Sullivan
Microsoft MVP: PowerShell

**Comment by devigned** [comment]
@vivsriaus and/or @ravbhatnagar do you have any thoughts on this item?

**Comment by vivsriaus** [comment]
The -OutputObjectFormat parameter has been removed in the latest powershell release. This was added because we made some breaking changes (unintentionally), and had to support the "legacy" output object format that some of our ARM cmdlets were returning originally. 

With regards to documentation gap, @ravbhatnagar has been working with our doc writer to improve coverage across our SDK tools (Powershell/CLI/SDK/etc) and will be able to provide an update when he's back from vacation next week.
